### PR TITLE
fix: Request Validator incorrectly refers to Kong format schema

### PR DESCRIPTION
**Pull request by lena-larionova**
### Description

The doc says the the Kong `format` schema validation is supported, which is incorrect. JSON Schema `format` validation is actually what's supported.

Fixes https://konghq.atlassian.net/browse/FTI-6196.

Note to reviewers: I fixed the title of the section referring to the "Kong format schema" and some minor cleanup to the page, including removing unnecessary output examples. 
However, I couldn't actually get these examples to work as documented in the example. The last step, validation, always fails for me- though this is probably out of scope of this PR and is a separate issue.

### Testing instructions

Preview link: 

### Checklist 

- [x] Review label added 
- [ ] [Conditional version tags](https://docs.konghq.com/contributing/conditional-rendering/#conditionally-render-content-by-version) added, if applicable.

### CODE REVIEW

File: app/_hub/kong-inc/request-validator/overview/_index.md [hunk 1]
```diff
@@ -419,13 +343,17 @@ In this example, use the plugin to validate a request's path parameter.
 
 4. In these step examples, validation ensures that `status_code` is a number and the body contains a parameter called `name`.
 
-   A proxy request with a non-numerical status code is blocked:
+   For example, if you try to send a request to an existing route with the path `/status/abc`:
```
Comment (cloudjumpercat) [inline app/_hub/kong-inc/request-validator/overview/_index.md:346]:
```suggestion
   For example, if you try to send a request to an existing route with the path `/status/123`:
```

**Review by cloudjumpercat — COMMENTED** [review]
Left one small suggestion

File: app/_hub/kong-inc/request-validator/overview/_index.md (hunk 1 above)
Comment (lena-larionova) [reply app/_hub/kong-inc/request-validator/overview/_index.md:346]: Ah - this one is right, the example is wrong. I'll fix that.

File: app/_hub/kong-inc/request-validator/overview/_index.md [hunk 2]
```diff
@@ -419,13 +343,17 @@ In this example, use the plugin to validate a request's path parameter.
...
 
-    ```
+    ```sh
     curl -i -X POST \
-    --url http://localhost:8000/status/abc \
+    --url http://localhost:8000/status/123 \
```
Comment (lena-larionova) [inline app/_hub/kong-inc/request-validator/overview/_index.md:350]:
```suggestion
    --url http://localhost:8000/status/abc \
```

**Review by cloudjumpercat — APPROVED** [review]

**Pull request merged**